thinc/layers/softmax.py

```diff
@@ -25,7 +25,7 @@ def Softmax(
         init=partial(init, init_W, init_b),
         dims={"nO": nO, "nI": nI},
         params={"W": None, "b": None},
-        attrs={"normalize_softmax": True},
+        attrs={"normalize_softmax": True, "softmax_temperature": 1.0},
```

Comment:
Ah it annoys me slightly that one attr is `XXX_softmax` and the other `softmax_XXX`. We can probably rename either of the two, as the first one has not been part of a release yet

Reply:
I think then having `softmax` at the beginning would be nicer.

Reply:
Renamed.